### PR TITLE
Option to export Data as HTML and JSON

### PR DESCRIPTION
**Pull request by mrbesen**
This should save users (and telegram servers), that want both: time, disk space and traffic.

This is how the menu in the general export looks:
grafik

I have exported everything with it. The resulting html and json files looked fine to me.

### CODE REVIEW

File: Telegram/SourceFiles/export/output/export_output_both.cpp [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	_html = std::make_shared<HtmlWriter>();
+	_json = std::make_shared<JsonWriter>();
+}
+
+Result BothWriter::start(
+		const Settings &settings,
+		const Environment &environment,
+		Stats *stats) {
+	Expects(_html != nullptr);
+	Expects(_html != nullptr);
```
Comment (fenderchamp) [inline Telegram/SourceFiles/export/output/export_output_both.cpp:27]: shouldn't one of these be _json?

Comment (mrbesen) [reply Telegram/SourceFiles/export/output/export_output_both.cpp]: yes, i fixed that.

**Comment by mrbesen** [comment]
Mentioned here: https://bugs.telegram.org/c/27133

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/commit/29bfe4338629be950e0dd32f5a77d00771fca4ea already merged another version?

**Comment by OTC15203** [comment]
OK HOW

**Pull request closed without merging**